packages/tools/src/tools/annotation/planarFreehandROITool/renderMethods.ts

```diff
@@ -128,6 +128,48 @@ function renderClosedContour(
   const allContours = [canvasPolyline, ...childContours];
   const polylineUID = '1';
 
+  if (annotation.data.contour.isPoint) {
```

Comment:
have a configuration on the planar free hand that is default false, something like displayOnePointAsCrosshairs and if that is true, you detect it is one point and that config is set tot display crosshairs